### ISSUE TITLE
Restrict login for users without mapped groups setting does nothing

### ISSUE THREAD

**Issue by joshbgosh10592** [zorn-v/nextcloud-social-login#134]
I just implemented Social Login (which is freaking awesome by the way, and thank you), but I'm trying to make it so that someone can create an account, but not actually log into it until I approve it somehow, even if that means adding them to a group.
I'm assuming the "Restrict login for users without mapped groups setting" flag in Social Login would do the trick, but I just tested it and my test account, which isn't a part of any groups, is able to log in and navigate around.

**Comment by zorn-v** [comment]
This setting only works when groups properly configured in provider settings.
For your request, maybe I should add option "create users with disabled account" ?

**Comment by joshbgosh10592** [comment]
I'm not sure I understand the functionality of that then, but the to create users with disabled account option sounds amazing! That should be able to complete exactly what I'm looking to do!

**Comment by zorn-v** [comment]
Check v2.3.0 - there is new option

You can enable/disable users in user list

**Comment by joshbgosh10592** [comment]
Awesome! I just updated (I went through a pretty big network re-org and took awhile to get everything back online.)
While it works pretty well, is there a way to get the email address, and maybe display name to populate into the new user? Or is that a system limitation?
Basically, when I created a new account, only "google-1283fkuy....." was displayed. If I have more than one pending request, I don't know who is who and what rights (if any) to grant.
